README.md

```diff
@@ -31,13 +39,21 @@ Feature  | Description
 ------------- | -------------
 [FirebaseListAdapter](https://firebaseui.com/docs/android/index.html)  | Binds a Firebase ref or query to a ListAdapter, which can populate a ListView or GridView.
 [FirebaseRecyclerViewAdapter](https://firebaseui.com/docs/android/index.html) | Binds a Firebase ref or query to a RecyclerView.
+[Firebase Auth UI](https://github.com/firebase/FirebaseUI-Android/tree/master/auth) | UI flows for Authentication
 
 ## iOS
 
 Feature  | Description
 ------------- | -------------
 [UITableViewDataSource](https://firebaseui.com/docs/ios/Classes/FirebaseTableViewDataSource.html)  | Binds a Firebase ref or query to a UITableView.
 [UICollectionViewDataSource](https://firebaseui.com/docs/ios/Classes/FirebaseCollectionViewDataSource.html) | Binds a Firebase ref or query to a UICollectionView.
+[Firebase Auth UI](https://github.com/firebase/FirebaseUI-iOS/tree/master/FirebaseAuthUI) | UI flows for Authentication
```

Comment:
These feature sections are relatively unmaintained, and I can see them falling out of sync with the iOS and Android source repos again in the near future, so I won't worry too much about them in this PR.